lms/static/sass/shared/_header.scss

```diff
@@ -357,7 +357,7 @@ header.global {
 // ====================
 
 // CASE: marketing/course discovery
-header.global-new {
+.header-global {
```

Comment:
This is the cause of the issues with headers. This file purposely had 2 distinct header styles, differentiated by ```.global`` and ```.global-new``` classes. By making both the same the earlier styles will be overridden by the later ones. I would recommend adding some classes so these 2 style blocks are distinct again.